package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "setup:docs": "./scripts/download-dependencies.sh",
     "setup:native": "./scripts/download-dependencies.sh && ./scripts/setup-react-android.sh",
-    "postinstall": "expo-yarn-workspaces check-workspace-dependencies",
+    "postinstall": "expo-yarn-workspaces check-workspace-dependencies && npx yarn-deduplicate",
```

Comment:
i'd rather add this as a workspace dependency and update it as needed, instead of using npx

Reply:
No problem, I will update the PR.

Reply:
Run deduplication before check-workspace-dependencies, which checks for correctness.